### ISSUE TITLE
Bugmeat in the wrong inventory category 

### ISSUE THREAD

**Issue by AmProsius** [AmProsius/gothic-1-community-patch#44]
Bugmeat should be in the category FOOD.

**Comment by AmProsius** [comment]
https://github.com/AmProsius/gothic-1-community-patch/blob/a8d2602eb8b6564733874644659d918e3fd7657d/scriptbase/_work/Data/Scripts/Content/Items/Food.d#L59-L63

changed to

```d
INSTANCE ItAt_Meatbug_01 (C_Item)
{
    name                =   "Bugmeat";

    mainflag            =   ITEM_KAT_FOOD;
```

**Comment by szapp** [comment]
This fix will be very similar to #49 and might be fast to implement.
